fluent-plugin-google-cloud.gemspec

```diff
@@ -21,10 +21,10 @@ eos
 
   gem.add_runtime_dependency 'fluentd', '~> 0.10'
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
-  gem.add_runtime_dependency 'google-api-client', '~> 0.17'
-  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.3', '>= 1.3.2'
-  gem.add_runtime_dependency 'googleauth', '~> 0.6'
-  gem.add_runtime_dependency 'grpc', '~> 1.0'
+  gem.add_runtime_dependency 'google-api-client', '~> 0.14'
+  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.2', '<= 1.2.3'
```

Comment:
Why the `<=` change?

Reply:
Because `google-cloud-logging` [v1.3.0](https://rubygems.org/gems/google-cloud-logging/versions/1.3.0) requires `google-gax` [v0.10.1](https://rubygems.org/gems/google-gax/versions/0.10.1) which requires `gRPC` `1.7.2`.

To verify the memory leak is not from `google-gax`, I tried the same `google-cloud-logging` and `google-gax` version with different `gRPC` version `v1.2.5` and `v1.4.5`. The memory leak exists in `v1.4.5`, but not `v1.2.5`.

Reply:
Why not `'< 1.3'`, then? Should we just make it `'~> 1.2.3'`?

Reply:
Changed to `'~> 1.2.3'`.